README.md

```diff
@@ -38,6 +38,9 @@ If using leveldb scripts, copy your database to a test location (default is `~/d
 ```
 cp -r ~/AppData/Roaming/legends-of-idleon/"Local Storage"/leveldb ~/dev/leveldb
 ```
+### Updating the dev build since the addition of idleon-data as a submodule
```

Comment:
Please add a blank line before and after this line to match the other headers. Could this be abbreviated to "Updating idleon-data"?